### PR TITLE
 homepage for particular restaurant

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Added homepage for particular restaurants

Live Preview: https://infallible-bell-91c11f.netlify.app/views/restaurant-home.html

### CODE REVIEW

File: public/css/restaurant-home.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    position: relative;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+    gap: 2rem;
+}
+
+
+
+.hotel-img{
```
Comment (iRohitGaur) [inline public/css/restaurant-home.css:32]: are you grouping the classes? I see some inconsistencies in spacing.

File: public/css/restaurant-home.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+.menu-lists{
+    flex: 7;
+    gap:1rem;
+}
+
+.radio{
+    accent-color: var(--radio-color);
+}
+
+.qg-flex-1{
```
Comment (amantulsyan35) [inline public/css/restaurant-home.css:105]: the naming is throwing me off a bit, can't seem to figure out what qg stands from

File: public/css/restaurant-home.css [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    position: -webkit-sticky;
+    position: sticky;
+    top: 15%;
+}
+
+.menu-section{
+    display: flex;
+    justify-content: space-between;
+    gap: 10px;
+    margin-top: 1;
```
Comment (iRohitGaur) [inline public/css/restaurant-home.css]: I think there should be a unit after 1 here

File: public/css/restaurant-home.css [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    }
+    
+     .discount-container{
+         height: 66vh;
+         gap:0;
+     }
+
+    .ui-card-list {
+      width: auto; 
+
```
Comment (iRohitGaur) [inline public/css/restaurant-home.css]: Inconsistent spacing

File: public/css/restaurant-home.css [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+    max-height: 14.375rem;
+    max-width: 21.875rem;
+    flex: 1;
+    object-fit: cover;
+}
+
+
+
+
+.hotelname{
```
Comment (amantulsyan35) [inline public/css/restaurant-home.css]: below for the hotel-list '-' is given but here '-' is not there, making the name a bit inconsistent

File: views/restaurant-home.html [hunk 6]
```diff
@@ -0,0 +1,429 @@
...
+             <div class="qg-flex-5  flex flex-column space-between gap-2 pt-1 pb-1">
+                 <div class="users flex flex-column gap-1 ">
+                     <span class="pb-01 hotelname ">Seven Kitchen Regels</span>
+                     <span class="hoteldesc">North-Indian, Mughlai, Tandoori.</span>
+                 </div>
+                
+                 <div class="user-dashboard flex flex-wrap flex-row  gap">
+                     <div class="flex flex-align-item-center gap">
+                         <div>
+                             <box-icon name='star' type='solid' color='#FFFFFF' ></box-icon>
```
Comment (iRohitGaur) [inline views/restaurant-home.html:80]: do not use inline styling

File: views/restaurant-home.html [hunk 7]
```diff
@@ -0,0 +1,429 @@
...
+                                    <label for="tabs-group4" class="tabs-title">Features</label>
+                                    <div class="tabs-content">
+                                       Lorem ipsum, dolor sit amet consectetur adipisicing elit. Culpa, nam nobis, impedit ipsam assumenda dolores molestias delectus cumque corporis enim dolorem incidunt nihil similique sint libero amet iste exercitationem eaque.
+                                    </div>
+                        
+                                   
+                                </div>
+                    
+                            </div>
+                            <div class="flex  pt-1 pb-1 flex-justify-end">
```
Comment (iRohitGaur) [inline views/restaurant-home.html:213]: class names can be more descriptive here

File: views/restaurant-home.html [hunk 8]
```diff
@@ -0,0 +1,429 @@
...
+                        vegetables and 2 seasoning.</span>
+                </div>
+                <div class="ui-card-list-button flex flex-column gap">
+                    <button
+                        class="btn-outline-primary btn btn-py-1 flex  ui-card-small-override-prop">
+                        <span class="icon-default-pl">Add To Cart</span>
+                        <span class="material-icons icon-sm">add</span>
+                    </button>
+                    <button
+                    class="btn-outline-primary btn btn-py-1 flex gap-1 ui-card-small-override-prop">
```
Comment (iRohitGaur) [inline views/restaurant-home.html:333]: what does this class do: "btn-py-1"?

suggestion: class names can be more descriptive

File: views/restaurant-home.html (hunk 7 above)
Comment (mehulsatardekar) [reply views/restaurant-home.html:213]: > class names can be more descriptive here

@iRohitGaur here pt-1 class describes padding-top and pb-1 padding bottom

**Comment by iRohitGaur** [comment]
issues

Filter section is too big. The font is larger than your main logo.

Suggestion: decrease the size of filter area a bit.

File: views/restaurant-home.html (hunk 8 above)
Comment (mehulsatardekar) [reply views/restaurant-home.html:333]: > what does this class do: "btn-py-1"?
> 
> suggestion: class names can be more descriptive

@iRohitGaur i have followed this naming convention from bootstrap and tailwind CSS. so here py describe padding everywhere on button and 1 is just a rem unit

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Needs some changes.

Good to merge with the updates as suggested.

File: public/css/restaurant-home.css (hunk 1 above)
Comment (mehulsatardekar) [reply public/css/restaurant-home.css:32]: nope, yeah will remove the unnecessary spacing

File: public/css/restaurant-home.css (hunk 2 above)
Comment (mehulsatardekar) [reply public/css/restaurant-home.css:105]: > the naming is throwing me off a bit, can't seem to figure out what qg stands from

@amantulsyan35 .**qg** it's just a prefix value, I have added cause of not to collide with the Component library.

File: views/restaurant-home.html (hunk 6 above)
Comment (mehulsatardekar) [reply views/restaurant-home.html:80]: > do not use inline styling

 yes Rohit, it is actually a boxicon icons styling which they provide by default. I tried changing color by CSS class. but it didn't work. if u notice these **type** and **color** are attributes probably they have implemented JS code for that. not sure.

**Comment by mehulsatardekar** [comment]
> issues
> 
> Filter section is too big. The font is larger than your main logo.
> 
> Suggestion: decrease the size of filter area a bit.

will do the changes

File: public/css/restaurant-home.css (hunk 3 above)
Comment (mehulsatardekar) [reply public/css/restaurant-home.css]: Done

File: public/css/restaurant-home.css (hunk 5 above)
Comment (mehulsatardekar) [reply public/css/restaurant-home.css]: will make the changes

File: public/css/restaurant-home.css [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+}
+.nav-zindex{
+    z-index: 999;
+}
+
+.discount-container{
+    height: 35vh;
+    background: var(--food-slider-bg);
+    position: relative;
+    padding: 10px;
```
Comment (varun13169) [inline public/css/restaurant-home.css:26]: Please use rems instead of px

File: public/css/restaurant-home.css [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+    align-items: center;
+    gap: 2rem;
+}
+
+
+
+.hotel-img{
+    height: 15.625rem;
+    width: 20.625rem;
+    border-radius: 7px;
```
Comment (varun13169) [inline public/css/restaurant-home.css:35]: Use rems instead of pxs

File: public/css/restaurant-home.css [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+    overflow: auto;
+    position: -webkit-sticky;
+    position: sticky;
+    top: 15%;
+}
+
+.menu-section{
+    display: flex;
+    justify-content: space-between;
+    gap: 10px;
```
Comment (varun13169) [inline public/css/restaurant-home.css:90]: Please use rems instead of px

File: public/css/restaurant-home.css [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+    }
+    .divider{
+        display: none;
+    }
+   
+    .filterbtn{
+        display: block;
+    }
+    .modal{
+        top:70px;
```
Comment (varun13169) [inline public/css/restaurant-home.css]: Please use rems instead of px

**Review by varun13169 — APPROVED** [review]

**Comment by varun13169** [comment]
Suggested minor changes, can be taken up in a follow up PR

File: public/css/restaurant-home.css (hunk 10 above)
Comment (mehulsatardekar) [reply public/css/restaurant-home.css:35]: @varun13169 if i am not wrong Vikram told in his session that if the sizing value is more than 10px then only use rems

**Comment by mehulsatardekar** [comment]
@iRohitGaur @amantulsyan35 @varun13169 thanks for the suggestion guys closing the PR with changes.

**Pull request merged**